website/content/docs/v0.10/Guides/disaster-recovery.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Disaster Recovery"
+description: "Procedure for snapshotting etcd database and recovering from catastrophic control plane failure."
+---
+
+`etcd` database backs Kubernetes control plane state, so if `etcd` service is unavailable
+Kubernetes control plane goes down, and the cluster is not recoverable until `etcd` is recovered with contents.
+`etcd` consistency model builds around consensus protocol Raft, so for highly-available control plane clusters,
+loss of one control plane doesn't impact cluster health.
+In general, `etcd` stays up as long as quorum nodes are up.
+For three control plane node Talos cluster, this means that the cluster tolerates a failure of any single node,
+but losing more than one node leads to complete cluster failure.
```

Comment:
This sounds just a touch _too_ dire.  Maybe: "losing more than one node at the same time" or "complete loss of service"?